packages/@unimodules/react-native-adapter/src/nativeEmitters.ts

```diff
@@ -1,4 +1,4 @@
-import RCTDeviceEventEmitter from 'react-native/Libraries/EventEmitter/RCTDeviceEventEmitter';
-import RCTEventEmitter from 'react-native/Libraries/vendor/emitter/EventEmitter';
+import RCTDeviceEventEmitter from 'react-native-web/dist/vendor/react-native/NativeEventEmitter/RCTDeviceEventEmitter';
+import RCTEventEmitter from 'react-native-web/dist/vendor/react-native/emitter/EventEmitter';
```

Comment:
I don't think we should have web-only imports in our universal `.ts` files... instead define `nativeEmitters.web.ts`, and if we need to, make `nativeEmitters.ts` with some shims like `export const RCTEventEmitter = {};` (maybe there is a better shim that throws clearer errors, but this basic idea still stands).

Reply:
Agreed, in this particular case however, RNW creates an emitter that works without the browser.